neo/rawio/axonarawio.py

```diff
@@ -558,6 +557,20 @@ def get_active_tetrode(self):
                     tetrode_id = int(key.strip("collectMask_"))
                     active_tetrodes.append(tetrode_id)
         return active_tetrodes
+    
+    def get_active_channels(self):
```

Comment:
public or private? what do you think?

Reply:
I advice to make things private by default and move away from it when you have a good reason to.

Reply:
That's how I was leaning too. I don't see why the user would run this function.

Reply:
Yes, I agree with making them private